test/Jamfile.v2

```diff
@@ -22,9 +24,9 @@ test-suite iterator
     [ run zip_iterator_test2_fusion_vector.cpp ]
     [ run zip_iterator_test2_fusion_list.cpp ]
 #    [ run zip_iterator_test2_fusion_deque.cpp ] // See bug report for fusion https://svn.boost.org/trac/boost/ticket/11572
-    [ run zip_iterator_test_fusion.cpp ]
-    [ run zip_iterator_test_std_tuple.cpp ]
-    [ run zip_iterator_test_std_pair.cpp ]
+    [ run zip_iterator_test_fusion.cpp : : : <source>/boost/assign//boost_assign ]
```

Comment:
Why are these `<source>` dependencies? Shouldn't they be normal "linking" dependencies?